src/DataFixtures/UserFixtures.php

```diff
@@ -120,17 +120,17 @@ private function getUserData(): array
             ],
             [
                 'displayname' => 'John Doe',
-                'username' => 'john_user',
+                'username' => 'john_editor',
                 'password' => $this->append ? Str::generatePassword(10) : 'john%1',
                 'email' => 'john_user@example.org',
-                'roles' => ['ROLE_USER'],
+                'roles' => ['ROLE_EDITOR'],
             ],
             [
-                'displayname' => 'Developer',
-                'username' => 'developer',
-                'password' => $this->append ? Str::generatePassword(10) : 'developer%1',
-                'email' => 'developer@example.org',
-                'roles' => ['ROLE_DEVELOPER'],
+                'displayname' => 'Eddie Enduser',
+                'username' => 'eddie',
+                'password' => $this->append ? Str::generatePassword(10) : 'eddie%1',
+                'email' => 'eddie@example.org',
+                'roles' => ['ROLE_USER'],
             ],
         ];
     }
```

Comment:
@bobdenotter General question: given that now all roles (not just `ROLE_ADMIN`) have access to the backend, should all these accounts be enabled by default? Can it be an issue in the event somebody setting up a site forgets to delete all the fixture users?

Reply:
I'm assuming one of these happens: 

 - Somebody runs `make db-reset` (us developing, or automated tests) and `admin` has `admin%1` as password. 
 - Somebody runs `bin/console bolt:setup`, and the first `developer` user gets setup. Then, the fixtures will add these users, but with a proper, encoded password. 

So, in practice, this shouldn't be a problem.

Reply:
Ah, good point! Thanks! 👍